tests/test_data/simple-jester.py

```diff
@@ -0,0 +1,27 @@
+r = []
```

Comment:
We should have a comment at the top explaining what this file does.

Reply:
Done